src/content/docs/vulnerability-management/change-vulnerability-status.mdx

```diff
@@ -0,0 +1,203 @@
+        src={vmIgnoreBulkDetails}
+    />
+
+1. Fill out the form to explain the reasoning behind changing this vulnerability to Ignored. Include a time period that the vulnerability should be ignored.
+
+        <img
+            title="Vulnerabilities details form "
+            alt="An image showing the ignore form on vulnerability instance details page"
+            src={vmIgnoreBulkForm}
+        />
+    </Collapser>
+</CollapserGroup>
+
+## Change Ignored status to Affected [#bulk-manage]
+
+It's good practice to review ignored vulnerabilities and validate whether they should stay ignored. Changing the status from Ignored to Affected reintroduces the vulnerability back into default views andcounts on this entity, but other instances of this vulnerability on other entities will still remain ignored.
+
+<CollapserGroup>
+    <Collapser
+        id="remove-ignored"
+        title="Remove ignored status for a specific entity"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select the entity you want to update) > **Vulnerability Management** > Click the **Vulnerabilities** tab.
+1. View all your ignored vulnerabilities by updating the filter from `Status = Affected` to `Status = Ignored`. 
+
+    <img
+    title="Vulnerabilities tab scoped to a specific entity"
+    alt="An image showing the scoped vulnerabilities tab of Vulnerability Management with filter tab open"
+    src={vmAffectedEntityVulnList}
+    />
+
+1. Choose the vulnerability that you want to update, then click **Change to affected**.
+
+    <img
+        title="Vulnerabilities tab scoped to a specific entity and only ignored instances"
+        alt="An image showing the scoped vulnerabilities tab of Vulnerability Management filtered to only ignored instances"
+        src={vmAffectedEntityFiltered}
+    />
+
+1. Fill out the form to explain the reasoning behind changing this vulnerability to Affected.
+        
+    <img
+        title="Vulnerability instance details form"
+        alt="An image showing the affected form on vulnerability instance details page"
+        src={vmAffectedEntityForm}
+    />
+
+    </Collapser>
+    <Collapser
+        id="bulk-change-status"
+        title="Remove Ignored status across multiple instances"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > Click the **Vulnerabilities** tab 
+1. From the settings cog, select the **Ignored** checkbox. This adds an ignore column to your vulnerability table.
+1. Select a vulnerability with an ignored count greater than 0. 
+
+    <img
+        title="Vulnerabilities tab with ignored column added"
+        alt="An image showing the all entities vulnerabilities tab with an ignore column added"
+        src={vmAffectedBulkVulnList}
+    />
+
+You can also bulk manage your Ignored vulnerabilities with the filter bar. This method is better if you X:
```

Comment:
This sentence and restarting of the steps really doesn't make sense to me?

Reply:
ah ok! that makes sense. i'll merge 'em in then